src/Okta.Sdk/Generated/IGroupClient.Generated.cs

```diff
@@ -57,15 +56,15 @@ public partial interface IGroupClient : IOktaClient
         /// <param name="ruleId"></param>
         /// <param name="removeUsers"></param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns></returns>
+        /// <returns>A Task that represents the asynchronous operation.</returns>        
         Task DeleteRuleAsync(string ruleId, bool? removeUsers = null, CancellationToken cancellationToken = default(CancellationToken));
 
         /// <summary>
         /// 
         /// </summary>
         /// <param name="ruleId"></param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns></returns>
+        /// <returns>A Task that represents the asynchronous operation.</returns>        
```

Comment:
For all cases of trailing whitespace in any of the `*.Generated.cs` files - it's possible to remove this, but it takes a lot of super-fine manual tweaking of the template files. It'll be cleaned up eventually but for now it's low-prio and doesn't cause any problems.